platform/feature-api/communication/remote-pod/src/remote.ts

```diff
@@ -74,13 +74,20 @@ type InfoEndpoint = ObjectEndpointSpec<{
     getVersion(): ValueEndpointSpec<string>;
 }>;
 
-type NetworkEndpoint = ObjectEndpointSpec<{
-    httpPost(
-        url: string,
-        body: string,
+type EndpointEndpoint = ObjectEndpointSpec<{
+    send(
+        endpointId: string,
```

Comment:
As this set of arguments is used repeatedly, wouldn't it make sense to create a `Record` for them?
There's the basic:
endpointId+contentType+payload
And then there are additional `authorization` and `featureIdToken`. That might be a (small) class hierarchy, right?

Reply:
I'm going to solve this, and leave it for a possible refinement.